modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1813,16 +1787,14 @@ private CompletableFuture<Void> handleChangePendingAssignmentEvent(
                         }
 
                         return setTablesPartitionCountersForRebalance(replicaGrpId, revision, pendingAssignments.force())
-                                .thenCompose(r ->
-                                        handleChangePendingAssignmentEvent(
-                                                replicaGrpId,
-                                                table,
-                                                pendingAssignments,
-                                                stableAssignments == null ? emptySet() : stableAssignments.nodes(),
-                                                revision,
-                                                isRecovery
-                                        )
-                                )
+                                .thenCompose(r -> handleChangePendingAssignmentEvent(
+                                        replicaGrpId,
+                                        table,
+                                        pendingAssignments,
```

Comment:
I'd rather have same order everywhere: stable then pending.

Reply:
Nice, I didn't notice this